### DISCUSSION TITLE
Skyline plugin has panicked error code  2-0069

### DISCUSSION THREAD

**Discussion by Flury2** [Raytwo/ARCropolis#452] [Issues] [answered]
System Firmware Version
14.1.2

Atmosphere Version
1.3.2

Smash Version
13.0.0

ARCropolis Version
3.9.0

Can the crash be consistently reproduced?
Yes

At what moment does the crash happen?
Starting up the game

Details:
The problem just started when i updated both my arcropolis and skyline any thoughts?

![1](https://user-images.githubusercontent.com/94097782/200448558-17d3b06b-5e62-4bfa-966b-786299ed8af3.jpg)
![2](https://user-images.githubusercontent.com/94097782/200448563-132515b7-25a7-47ac-bc0b-e617d2c65e11.jpg)

**Comment by ThatNintendoNerd** [comment, marked as answer]
ARCropolis assumes the user has Ver. 13.0.1 installed due to numerous instances of hardcoding executable offsets from that version instead of deriving offsets to work on multiple game versions. Update your game and the problem should be resolved.

**Comment by Valdens** [comment]
Getting the same issue on a fresh install with the correctly updated Smash patch - the only difference in my case from OP is that I'm on system firmware version 15.0.1.
